doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-kernel.git
+  $ cd acrn-kernel
+  $ git checkout acrn-2019w31.5-140000p
+  $ cp kernel_config_sos .config
+  $ make menuconfig   ## Don't do anything, just save and quit to set the default config items
+  $ make bzImage && make modules
+  $ make modules_install INSTALL_MOD_PATH=$PWD
+
+Update the Service VM kernel
+============================
+Follow the steps to enable the new Service VM kernel and its modules::
+
+  # mount /dev/sda1 /mnt
+  # scp <host machine>:~/acrn-kernel/arch/x86/boot/bzImage /mnt/EFI/org.clearlinux/
+  # sed -i "s/$(sed -n '2p' /mnt/loader/entries/acrn.conf | cut -d'/' -f4)/bzImage/" /mnt/loader/entries/acrn.conf
+  # scp -r <host machine>:~/acrn-kernel/lib/modules/* /usr/lib/modules/
+  # sync && umount /mnt
+  # reboot
+
+How to update Preempt_RT kernel
+*******************************
+In this section, we will learn to update the Preempt_RT kernel by two methods:
+
+- Rebuild from acrn-kernel branch tree.
+- Install Preempt_RT kernel from Clear Linux bundles.
+
+You could try either of them to get the Preempt_RT kernel and then copy it onto NVMe disk.
+
+How to rebuild the Preempt_RT kernel from acrn-kernel branch tree
+=================================================================
+#. Build the Preempt_RT kernel on your host machine (non-root privilege)::
+
+   $ cd ~/acrn-kernel
+   $ git reset --hard
+   $ git clean -df
+   $ git checkout acrn-2019w31.5-143000p
+   $ cp x86-64_defconfig .config
+   $ make bzImage && make modules                 # Press 'Enter' to use default configuration
+   $ make modules_install INSTALL_MOD_PATH=$PWD
+
+#. Copy the Preempt_RT kernel and modules to the target platform
+   (pass-though NVME to CL-RT VM as the rootfs dev)::
+
+   # mount /dev/nvme0n1p3 /mnt
+   # scp <host machine>:~/acrn-kernel/arch/x86/boot/bzImage /mnt/boot/bzImage.preempt-rt
+   # scp -r <host machine>:~/acrn-kernel/lib/modules/* /mnt/lib/modules/
+   # sync && umount /mnt
+
+How to install the Preempt_RT kernel from bundle
+================================================
+#. Add ``kernel-lts2018-preempt-rt`` bundle from Clear Linux (root privilege)::
+
+   # swupd bundle-add kernel-lts2018-preempt-rt
+
+#. Copy Preempt_RT module to the NVMe disk:
+
+   .. code-block:: console
+
+      # mount /dev/nvme0n1p3 /mnt
+      # ls /usr/lib/modules/
+      4.19.59-13.iot-lts2018-preempt-rt/     4.19.36-48.iot-lts2018/           4.19.36-48.iot-lts2018-sos/        5.0.14-753.native/
+      # cp -r /usr/lib/modules/4.19.59-13.iot-lts2018-preempt-rt /mnt/lib/modules/
+      # sync && umount /mnt
+
+
+How to change launch script for CL-RT VM
+****************************************
+In this section, we will learn to launch the CL-RT VM.
+
+Create the script for CL-RT VM
+==============================
+#. Create a ``launch_hard_rt_vm.sh`` script:
+
+   .. code-block:: console
```

Comment:
There is a [`launch_hard_rt_vm.sh`](https://github.com/projectacrn/acrn-hypervisor/blob/master/devicemodel/samples/nuc/launch_hard_rt_vm.sh) script in our repo already, could we re-use it or consolidate the two?

Reply:
I've changed the step to "Download [`launch_rtvm`](https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/scripts/launch_hard_rt_vm.sh) script to the Service VM", since this one is already customized as an example RTVM script for this introduction so that user no need to modify anymore just download and use it.